GitUI/CommitInfo/CommitInfo.Designer.cs

```diff
@@ -186,6 +187,7 @@ private void InitializeComponent()
             this._RevisionHeader.ContentsResized += new System.Windows.Forms.ContentsResizedEventHandler(this._RevisionHeader_ContentsResized);
             this._RevisionHeader.LinkClicked += new System.Windows.Forms.LinkClickedEventHandler(this.RevisionInfoLinkClicked);
             this._RevisionHeader.MouseDown += new System.Windows.Forms.MouseEventHandler(this._RevisionHeader_MouseDown);
+            this._RevisionHeader.KeyDown += new System.Windows.Forms.KeyEventHandler(this.RichTextBox_KeyEvent);
```

Comment:
Please rename `RichTextBox_KeyEvent` to `RichTextBox_KeyDown` to match the event it handles

Reply:
Done